system/jlib/jsocket.cpp

```diff
@@ -4869,8 +4874,7 @@ class CSocketEpollThread: public CSocketBaseThread
             // and more than one thread is present
             unsigned n = items.ordinality();
             if (n == 0 && numThreads > 1)
-                terminating = true;
-            triggerselect();
+                stop(false);
```

Comment:
Do you still want to call triggerselect() if not terminating?

Reply:
I suppose yes to make it like it was before.  I will revert it back.